ci/tests/harness.h

```diff
@@ -0,0 +1,34 @@
+/** A simple harness for the C benchmarks
+ *
+ * It is included by every benchmark file and provides the time measuring,
+ * as well as standardized command-line argument parsing.
+ */
+
+#include <sys/time.h>
+
+unsigned long microseconds() {
+  // Not monotonic
+  struct timeval t;
+  gettimeofday(&t, NULL);
+  return (t.tv_sec * 1000 * 1000) + t.tv_usec;
+};
+
+void parse_argv(int argc,
```

Comment:
It seems  this function is called in only one place [1], with static arguments?

[1] https://github.com/capablevms/bdwgc/pull/18/files#diff-62547593811bc728a95eb6141642a30dd4d5ff5a243071085b1aeaab696035acR422

Reply:
It is only called in one place @0152la . This is imported code and I did not want to change it unless there was a need to do so.